### DISCUSSION TITLE
English documentation

### DISCUSSION THREAD

**Discussion by maaamcube** [cherry-embedded/CherryUSB#262] [Ideas]
Hello,
Is there an english documentation of CherryUSB ? if not, when it will be released ?
Thanks.

**Comment by sakumisu** [comment]
What kind of content do you need？API or other? A lot of the contents are about usb provided by usb official, and we translated those into chinese,  i think those you do not need because they are english.

**Comment by maaamcube** [comment]
@sakumisu everything if it's possible because i don't undrestand chinese.

**Reply by sakumisu** [reply]
> @sakumisu everything if it's possible because i don't undrestand chinese.

Sorry, but most of them are from usb org and i translated into chinese, so you can get from usb org directly.

**Reply by maaamcube** [reply]
> > @sakumisu everything if it's possible because i don't undrestand chinese.
> 
> Sorry, but most of them are from usb org and i translated into chinese, so you can get from usb org directly.

Even the documentation related to library ?

**Reply by sakumisu** [reply]
> > > @sakumisu everything if it's possible because i don't undrestand chinese.
> > 
> > 
> > Sorry, but most of them are from usb org and i translated into chinese, so you can get from usb org directly.
> 
> Even the documentation related to library ?

Only cherryusb api related, others like what is usb, usb class and so on are related to doc provided by usb org.

**Reply by maaamcube** [reply]
ok then

**Comment by sakumisu** [comment]
done
